### ISSUE TITLE
adding ge kretz 2d data access

### ISSUE THREAD

**Issue by gordon-n-stevenson** [KitwareMedical/ITKUltrasound#174]
Hi 👋

We've been using the 2d image classes in itkultrasound to resample polar 2d data from a ge voluson system. Keen to share and add this IO functionality. It seems better to extend itkultrasound than build another module would this be sensible?

Happy to put a pull request together

**Comment by dzenanz** [comment]
Yes, of course! Contributions are welcome. Remote modules generally follow the same [contribution process](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md) as the main ITK repository.
